app/screens/contacts-screen/contacts.tsx

```diff
@@ -113,16 +112,12 @@ export const ContactsScreen: ScreenType = ({ navigation }: Props) => {
         }
       }
     `,
-    { skip: !hasToken },
+    {
+      skip: !hasToken,
+      fetchPolicy: "cache-and-network",
```

Comment:
Without `cache-and-network`, once the data is persisted, Apollo does not go back to the server (even when the component re-renders fresh!). 

I think we need `cache-and-network` in most cases.